src/facade/dragonfly_listener.cc

```diff
@@ -77,19 +80,24 @@ static SSL_CTX* CreateSslCntx() {
     if (!tls_cert_file.empty()) {
       // TO connect with redis-cli you need both tls-key-file and tls-cert-file
       // loaded. Use `redis-cli --tls -p 6380 --insecure  PING` to test
-
       CHECK_EQ(1, SSL_CTX_use_certificate_chain_file(ctx, tls_cert_file.c_str()));
     }
+
+    const auto tls_ca_cert_file = GetFlag(FLAGS_tls_ca_cert_file);
```

Comment:
Why did you add this?
Did we have a task on this or you just saw this was missing and wanted to add it ?
I see redis has tls-auth-clients config, if this config is set then validation is done.
I suggest adding tls_auth_client flag and if it is true you should make sure that one of the flags dir / file is set

Reply:
if both flags are empty, it's equivalent to `tls_auth_client=false` 
if some of the flags are not empty, it's equivalent to `tls_auth_client=true` 

Why do you need the state where `tls_auth_client=false` and some of the flags are not empty?

Reply:
just have similar config as redis

Reply:
yes but in this particular case it's a redundant flag. I am all in when it comes to be as close to redis as possible but when it's tangible to improve I opt in for that.